docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -1,283 +1,258 @@
 
-![Elastic Beanstalk Base Configuration](images/EBBaseConfig.png)
+- First, make sure `Platform` is set to `Preconfigured platform`, with the dropdown set to `Docker`.
+- Next, change the `Application code` setting to `Upload your code`, then click the `Upload` button. You'll see a modal like this:
 
-This will run our Metabase application using  [Docker](https://www.docker.com) under the hood.
+![Elastic Beanstalk Base Configuration](images/EBUploadYourCode.png)
 
-If you reached this screen from the Metabase start page at [metabase.com/start/aws/](https://metabase.com/start/aws/) the application code settings will have already been set and you don't need to do anything. This will use the official Metabase Docker image which is [published on Dockerhub](https://hub.docker.com/r/metabase/metabase/).
+The modal should be pre-populated with the correct settings if you used the quick launch URL at the top of these instructions. All you need to do is push the `Upload` button in this modal.
 
-When your environment type settings look like the above then go ahead and click `Review and launch`.
+These settings will run the Metabase application using [Docker](https://www.docker.com) under the hood, using the official Metabase Docker image which is [published on Dockerhub](https://hub.docker.com/r/metabase/metabase/).
 
+You can now go ahead and click `Review and launch`.
 
-##### Enabling enhanced health checks
+#### Enabling enhanced health checks
 
 You will need to enable enhanced health checks for your Beanstalk Monitoring.
 
-Click on the `modify` link under the Monitoring section as below.
+Click on the `Modify` link under the Monitoring section as seen here:
+
 ![Elastic Beanstalk Monitoring](images/EBMonitoringSelect.png)
 
 Then make sure enhanced health checks are enabled. This is a free option, unless you later add specific metrics to CloudWatch.
 
 ![Elastic Beanstalk Monitoring Settings](images/EBMonitoringSettings.png)
 
-##### Configuring RDS for Metabase
+#### Configuring RDS for Metabase
 
-To run Metabase in a cloud environment of any kind we highly recommend using an independent database server with high availability such as Amazon RDS.  So for standard deployments we will choose to create an RDS instance with our Elastic Beanstalk application.
+To run Metabase in a cloud environment of any kind we highly recommend using an independent database server with high availability such as Amazon RDS. So for standard deployments we will choose to create an RDS instance with our Elastic Beanstalk application.
 
+_NOTE: It's possible to skip this step if you wish. However, this will force Metabase to use a local H2 database file on your application server, and there will be no way to backup and maintain that database. **When your instance is restarted for any reason you'll lose all your Metabase data**. If you are just doing a quick trial of Metabase that may be okay, but otherwise we recommend against it._
```

Comment:
This does not seem to be the case (anymore?). When I click "Create app" without filling in RDS username/password, I get the following: 
![image](https://user-images.githubusercontent.com/23798/66212904-0a739600-e6bf-11e9-9a7c-9078528746f5.png)